src/full/Agda/TypeChecking/Serialise.hs

```diff
@@ -82,7 +82,7 @@ import Agda.Utils.Impossible
 -- 32-bit machines). Word64 does not have these problems.
 
 currentInterfaceVersion :: Word64
-currentInterfaceVersion = 20221027 * 10 + 0
+currentInterfaceVersion = 20221028 * 10 + 0
```

Comment:
Maybe the bump isn't necessary in this PR since only a constructor was renamed (but is still serialized the same).  
Anyway, the bump doesn't hurt, so, ignore my comment.